src/components/RegionList/RegionListComponent.tsx

```diff
@@ -22,9 +21,10 @@ export class RegionListComponent extends React.Component<WidgetProps> {
     @observable height: number = 0;
 
     private static readonly LOCK_COLUMN_DEFAULT_WIDTH = 25;
+    private static readonly FOCUS_COLUMN_DEFAULT_WIDTH = 25;
```

Comment:
could we replace these both with `ACTION_COLUMN_DEFAULT_WIDTH`?

Reply:
Fixed